agent/src/softwarecontaineragent.cpp

```diff
@@ -261,8 +303,38 @@ void SoftwareContainerAgent::setGatewayConfigs(const uint32_t &containerID, cons
     }
 }
 
-bool SoftwareContainerAgent::setCapabilities(const uint32_t &containerID, const std::vector<std::string> &capabilities)
+void SoftwareContainerAgent::setGatewayConfigs(const uint32_t &containerID,
```

Comment:
I have done this, but feel like it may require even more changes to the softwarecontainer class, so let's see if this is sufficient. Changes in next upload.